go/config/config.go

```diff
@@ -129,6 +129,7 @@ type Configuration struct {
 	MySQLOrchestratorReadTimeoutSeconds        int      // Number of seconds before backend mysql read operation is aborted (driver-side)
 	MySQLDiscoveryReadTimeoutSeconds           int      // Number of seconds before topology mysql read operation is aborted (driver-side). Used for discovery queries.
 	MySQLTopologyReadTimeoutSeconds            int      // Number of seconds before topology mysql read operation is aborted (driver-side). Used for all but discovery queries.
+	MySQLConnectionLifetime                    int      // Number of seconds the mysql driver will keep database connection alive before recycling it
```

Comment:
Can you please rename the variable to `MySQLConnectionLifetimeSeconds`?

Reply:
Done.